Sources/X509/ExtensionsBuilder.swift

```diff
@@ -41,42 +41,69 @@
+    public static func buildExpression<Extension: CertificateExtensionConvertible>(_ expression: Extension) -> Result<Certificate.Extensions, any Error> {
+        Result {
+            try Certificate.Extensions([expression.makeCertificateExtension()])
+        }
+    }
+    
+    @inlinable
+    public static func buildExpression(_ expression: Certificate.Extensions) -> Result<Certificate.Extensions, any Error> {
+        .success(expression)
+    }
+    
+    @inlinable
+    public static func buildExpression() -> Result<Certificate.Extensions, any Error> {
+        .success(Certificate.Extensions())
+    }
+    
+    @inlinable
+    public static func buildBlock() -> Result<Certificate.Extensions, any Error> {
+        .success(Certificate.Extensions())
     }
 
     @inlinable
-    public static func buildBlock(_ components: Certificate.Extensions...) -> Certificate.Extensions {
-        Certificate.Extensions(components.lazy.flatMap { $0 })
+    public static func buildBlock(_ components: Result<Certificate.Extensions, any Error>...) -> Result<Certificate.Extensions, any Error> {
+        Result {
+            Certificate.Extensions(try components.lazy.flatMap { try $0.get() })
+        }
     }
 
     @inlinable
-    public static func buildOptional(_ component: Certificate.Extensions?) -> Certificate.Extensions {
-        component ?? Certificate.Extensions([])
+    public static func buildOptional(_ component: Result<Certificate.Extensions, any Error>?) -> Result<Certificate.Extensions, any Error> {
+        component ?? .success(Certificate.Extensions([]))
     }
 
     @inlinable
-    public static func buildEither(first component: Certificate.Extensions) -> Certificate.Extensions {
+    public static func buildEither(first component: Result<Certificate.Extensions, any Error>) -> Result<Certificate.Extensions, any Error> {
         component
     }
 
     @inlinable
-    public static func buildEither(second component: Certificate.Extensions) -> Certificate.Extensions {
+    public static func buildEither(second component: Result<Certificate.Extensions, any Error>) -> Result<Certificate.Extensions, any Error> {
         component
     }
 
     @inlinable
-    public static func buildArray(_ components: [Certificate.Extensions]) -> Certificate.Extensions {
-        Certificate.Extensions(components.lazy.flatMap { $0 })
+    public static func buildArray(_ components: [Result<Certificate.Extensions, any Error>]) -> Result<Certificate.Extensions, any Error> {
+        Result {
+            Certificate.Extensions(try components.lazy.flatMap { try $0.get() })
+        }
     }
 
     @inlinable
-    public static func buildLimitedAvailability(_ component: Certificate.Extensions) -> Certificate.Extensions {
+    public static func buildLimitedAvailability(_ component: Result<Certificate.Extensions, any Error>) -> Result<Certificate.Extensions, any Error> {
         component
     }
 }
 
+
+func foo() -> Certificate.Extensions {
```

Comment:
This seems likely to be left over from development

Reply:
Good catch